src/snowflake/cli/plugins/lsp/models/context.py

```diff
@@ -0,0 +1,95 @@
+# coding: utf-8
+
+"""
+    LSP Contracts
+
+    No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+
+    The version of the OpenAPI document: 1.0.0
+    Generated by OpenAPI Generator (https://openapi-generator.tech)
+
+    Do not edit the class manually.
+"""  # noqa: E501
+
+
+from __future__ import annotations
+import pprint
+import re  # noqa: F401
+import json
+
+from pydantic import BaseModel, ConfigDict, StrictStr
+from typing import Any, ClassVar, Dict, List, Optional
+from snowflake.cli.plugins.lsp.models.connection import Connection
```

Comment:
This is properly wired within the project thanks to `-p packageName=snowflake.cli.plugins.lsp`